crates/ruff_python_formatter/src/type_param/type_params.rs

```diff
@@ -1,6 +1,6 @@
 use ruff_formatter::write;
 use ruff_formatter::FormatResult;
-use ruff_python_ast::node::AstNode;
+use ruff_python_ast::AstNode;
 use ruff_python_ast::TypeParams;
```

Comment:
Here and elsewhere, many of these could be combined now, but I know it's hard to do automatically.